project/.github/workflows/ci.yml

```
@@ -83,7 +83,7 @@ jobs:
       # - Create GitHub release
       # - Publish to PyPI
       - name: Python Semantic Release
-        uses: relekang/python-semantic-release@v7.27.1
+        uses: relekang/python-semantic-release@master
```

I'd rather keep the versions of my github actions pinned personnally:

```suggestion
        uses: relekang/python-semantic-release@v7.32.2
```

I'm using Renovate to upgrade them, so it's usually not a problem very long on the generated projects. 

It would be nice to auto-update them in the templated part here though 🤔 Let me check the docs if it's possible to tell Renovate how to update things in subdirectories

---

Ok, I didn't know much about Renovate, sorry. 😇 Thank you for your swift reply.